mlflow/metrics/utils/model_utils.py

```diff
@@ -0,0 +1,73 @@
+import asyncio
+import os
+import re
+
+from mlflow.exceptions import MlflowException
+from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE
+
+ROUTE_TYPE = "llm/v1/completions"
+
+
+# TODO: improve this name
+def score_model_on_payload(model_uri, payload):
+    """Call the model identified by the given uri with the given payload."""
+
+    prefix, suffix = _parse_model_uri(model_uri)
```

Comment:
We should be using urllib to parse URIs, not regex splits. I think we have other utils for this in MLflow. Can we reuse them?